prov/verbs/src/verbs_eq.c

```diff
@@ -689,13 +689,31 @@ vrb_eq_xrc_rej_event(struct vrb_eq *eq, struct rdma_cm_event *cma_event)
 	return state == VRB_XRC_ORIG_CONNECTING ? FI_SUCCESS : -FI_EAGAIN;
 }
 
+/* Caller must hold eq:lock */
+static inline int
+vrb_eq_xrc_connect_retry(struct vrb_xrc_ep *ep,
+			 struct rdma_cm_event *cma_event, int *acked)
+{
+	*acked = 1;
+	rdma_ack_cm_event(cma_event);
```

Comment:
Why is the acked parameter needed?  It is only set to 1.
Can we assume that if this function returns successfully, then acked is 1.  Otherwise, if the function returns with an error then acked is 0.

Reply:
Thanks. The ACK was just left in as an inline sub-function was created to capture the retry logic for readability; i.e. it is only required for the retry.  As the function is an in-line function, I expect there is no difference in keeping the setting here or setting it outside of this grouping. However, I did not look at output to verify that